src/main/sensors/battery.c

```diff
@@ -114,6 +116,10 @@ PG_RESET_TEMPLATE(batteryConfig_t, batteryConfig,
     .consumptionWarningPercentage = 10,
     .vbathysteresis = 1,
 
+	.vbat_warn_delay = 0,
+	.vbat_crit_delay = 0,
+	.use_vbat_unfiltered = 0,
+	.vbat_high_hysteresis = 1,
```

Comment:
Indentation is off here.

Reply:
Please use camelCase for variable names. (With the exception of variables used for CLI settings, where the name should be identical to the name used in the CLI setting, i.e. snake_case.)